llvm/utils/TableGen/DecoderEmitter.cpp

```diff
@@ -76,6 +78,12 @@ static cl::opt<SuppressLevel> DecoderEmitterSuppressDuplicates(
             "significantly reducing Table Duplications")),
     cl::init(SUPPRESSION_DISABLE), cl::cat(DisassemblerEmitterCat));
 
+static cl::opt<uint32_t>
+    NumToSkipSizeInBytes("num-to-skip-size",
+                         cl::desc("number of bytes to use for num-to-skip "
```

Comment:
The help message isn't really helpful to the user as it talks about some implementation detail. Option's name doesn't tell much either.

How about replacing this with a boolean option like `large-tables` or similar?

Reply:
Done